### PR TITLE
[FIX] account_fiscal_year_closing: fix load template for multi company

### PR DESCRIPTION
**Pull request by matteoopenf**

### CODE REVIEW

**Comment by matteoopenf** [comment]
@OCA/community-maintainers  is it possible to merge?

**Comment by matteoopenf** [comment]
@sergiocorato can you made a review?
Thanks

**Comment by simahawk** [comment]
> @OCA/community-maintainers is it possible to merge?

you need reviews 1st :wink:

File: account_fiscal_year_closing/models/account_fiscalyear_closing.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def _default_company_id(self):
     closing_template_id = fields.Many2one(
         comodel_name="account.fiscalyear.closing.template",
         string="Closing template",
-        domain="[('chart_template_ids', '=', chart_template_id)]",
```
Comment (simahawk) [inline account_fiscal_year_closing/models/account_fiscalyear_closing.py:93]: I have no idea and no context... silly question: don't you have to keep the original domain too?

Comment (matteoopenf) [reply account_fiscal_year_closing/models/account_fiscalyear_closing.py:93]: Hi @simahawk, with that domain the template has not show

How an you see there:
image

and I have alredy create the template there:
image

Comment (simahawk) [reply account_fiscal_year_closing/models/account_fiscalyear_closing.py:93]: fine :)

**Pull request closed without merging**